pkg/runner/cluster_k8s.go

```diff
@@ -339,6 +339,7 @@ func (c *ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow *rpc
 		}
 
 		for i := 0; i < g.Instances; i++ {
+			var gracePeriod int64 = 0
```

Comment:
just for curiosity, do we need to specify the var type here?, if not, maybe we can follow the same approach as the other var definitions to keep it more standard:

```go
gracePeriod := 0
```

Reply:
actually you are right. I have no idea why I did this to begin with 😓 
Fixing

Reply:
if do gracePeriod := 0 it will think that this is int, which is not ok considering that it is int64 😅

Reply:
thats true! 😄 
good catch!